modules/js/index.js

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable no-restricted-globals */
+/*  eslint linebreak-style: ["error", "unix"]   */
+/* eslint-disable no-undef */
+import { printErrorMsg, addBtn } from './utils.js';
+
+let books = JSON.parse(localStorage.getItem('books'));
+
+function dT() {
+  const now = luxon.DateTime.now();
+  const currentTime = document.querySelector('.date');
+  currentTime.innerHTML = now.toLocaleString(
+    luxon.DateTime.DATETIME_FULL_WITH_SECONDS,
+  );
+}
+
+setInterval(dT, 1000);
+
+class Book {
+  constructor(id, title, author) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+  }
+
+  addBook() {
+    const { id, title, author } = this;
+    const bookObj = { id, title, author };
+    books = JSON.parse(localStorage.getItem('books'));
+    if (title === '' || author === '') {
+      printErrorMsg('Please fill in all the fields');
+    } else if (books !== null) {
+      books.push(bookObj);
+      localStorage.setItem('books', JSON.stringify(books));
+      books = JSON.parse(localStorage.getItem('books'));
+      document.getElementById('title').value = '';
+      document.getElementById('author').value = '';
+    } else {
+      books = [];
+      books.push(bookObj);
+      localStorage.setItem('books', JSON.stringify(books));
+      books = JSON.parse(localStorage.getItem('books'));
+      document.getElementById('title').value = '';
+      document.getElementById('author').value = '';
+    }
+  }
+
+  removeBook() {
+    const { id } = this;
+    books = books.filter((book) => {
+      if (book.id !== id) {
+        return true;
+      }
+      return false;
+    });
+    localStorage.setItem('books', JSON.stringify(books));
+  }
```

Comment:
- [ ] In reference to the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md), please re-write the removeBook function using es6 syntax.